workers/unity/Assets/Playground/Config/MetricConfig.cs

```diff
@@ -0,0 +1,12 @@
+namespace Playground
+{
+    public delegate float CalculateLoadDelegate(float fps);
+
+    public static class MetricConfig
+    {
+        public static CalculateLoadDelegate CalculateLoad;
+
+        public static int MaxFpsSamples = 50;
+        public static int TimeBetweenMetricUpdatesSecs = 2;
```

Comment:
Probably want this to be a float to be in keeping with Unity time

Reply:
Agree, I prefer it to be a float as well.